parquet-hadoop/src/main/java/org/apache/parquet/hadoop/metadata/ColumnChunkMetaData.java

```diff
@@ -241,6 +280,8 @@ public PrimitiveType getPrimitiveType() {
    * @return the stats for this column
    */
   abstract public Statistics getStatistics();
+  
+  abstract public void decryptIfNeededed();
```

Comment:
The design seems to require this method to be only invoked inside this package. So please, make it _package private_.

You may have an empty implementation here so you don't need to add the empty implementations in the Int/Long classes.

Also, a bit too many "_ed_"s:
```suggestion
  abstract public void decryptIfNeeded();
```